engines/bastion_katello/app/assets/javascripts/bastion_katello/products/views/products.html

```diff
@@ -21,6 +21,10 @@ <h2 translate>Products</h2>
       <span translate>Repo Discovery</span>
     </button>
 
+    <a href="/katello/api/v2/organizations/{{current_organization}}/products.csv" class="btn btn-default">
```

Comment:
It may be better to use [`ng-href`](https://docs.angularjs.org/api/ng/directive/ngHref) here in case the link is somehow clicked on before `current_organization` is populated.  The `ng-href` directive prevents this.